docs/reference/changelog.md

```diff
@@ -28,6 +28,10 @@
   * `.transactions`
   * `.milestones`
   * `ContractingProcesses.milestones`
+* [#428](https://github.com/open-contracting/infrastructure/pull/428) - add objects:
+  * `Meeting`
+  * `lobbyingMeetings`
+  * `PublicOfficial`
```

Comment:
`lobbyingMeetings` is a field rather than an object. I think it's fine to only mention the field since the `Meeting` and `PublicOfficial` definitions are only used in that context.

```suggestion
* [#428](https://github.com/open-contracting/infrastructure/pull/428) - add `lobbyingMeetings` field.
```